cfme/tests/test_login.py

```diff
@@ -241,11 +242,12 @@ def test_credentials_change_password_with_special_characters():
     pass
 
 
-@pytest.mark.manual
 @pytest.mark.tier(3)
+@pytest.mark.ignore_stream("5.10")
 @test_requirements.multi_region
+@pytest.mark.long_running
 @pytest.mark.parametrize('context', [ViaUI])
-def test_multiregion_displayed_on_login(context):
+def test_multiregion_displayed_on_login(context, setup_multi_region_cluster, multi_region_cluster):
```

Comment:
@izapolsk I'm not seeing this test in your PRT run, can you specify it in the PR description?

Reply:
you probably looked for it in 5.10 PRT run. It's present only in 5.11.